beetsplug/alternatives.py

```diff
@@ -67,14 +80,36 @@ def __init__(self, plugin):
         parser = ArgumentParser()
         subparsers = parser.add_subparsers(prog=parser.prog + ' alt')
         subparsers.required = True
-        subparsers.dest = 'update'
+
         update = subparsers.add_parser('update')
         update.set_defaults(func=plugin.update)
-        update.add_argument('name')
+        update.add_argument('name', metavar='NAME')
         update.add_argument('--create', action='store_const',
                             dest='create', const=True)
         update.add_argument('--no-create', action='store_const',
                             dest='create', const=False)
+
+        list_tracks = subparsers.add_parser(
+            'list-tracks',
+            description="""
+                List all tracks that are currently part of an alternative
+                collection""",
```

Comment:
Does `argparse` strip the additional linebreak here?

Reply:
Yes. (I had to insert the linebreak because of flake8)